interface/ceed-basis.c

```diff
@@ -1059,6 +1174,32 @@ int CeedBasisGetGrad1D(CeedBasis basis, const CeedScalar **grad_1d) {
   return CEED_ERROR_SUCCESS;
 }
 
+/**
+  @brief Get divergence matrix of a CeedBasisHdiv
+  @param basis        CeedBasisHdiv
+  @param[out] div  Variable to store divergence matrix
+  @return An error code: 0 - success, otherwise - failure
+  @ref Advanced
+**/
```

Comment:
```suggestion
  @brief Get divergence matrix of a CeedBasisHdiv
  
  @param basis        CeedBasisHdiv
  @param[out] div  Variable to store divergence matrix
  
  @return An error code: 0 - success, otherwise - failure
  
  @ref Advanced
**/
```

Reply:
maybe just `CeedBasis` not `CeedBasisHdiv` ?!

Reply:
good call